tests/snmpsim/ceraos.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2281.1.20.2.2
```

Comment:
hi @guillemmateos 
I have taken it out of my commit, I've also shortened the sysobjectid to the generic Ceraos (1.3.6.1.4.1.2281) rather than the IP20C which you used.
I will be making PR shortly.